cmd/vicadmin/server.go

```diff
@@ -262,7 +263,7 @@ func (s *server) Authenticated(link string, handler func(http.ResponseWriter, *h
 		log.Infof("User with a valid auth cookie at %s is authenticated.", connectingAddr[0])
 		handler(w, r)
 	}
-	s.mux.Handle(link, gorillacontext.ClearHandler(http.HandlerFunc(authHandler)))
+	s.Handle(link, http.HandlerFunc(authHandler))
```

Comment:
https://github.com/vmware/vic/blob/master/cmd/vicadmin/server.go#L161
s.Handle will make a call to ClearHandler with the httlHandler.